src/webgpu/shader/execution/expression/call/builtin/textureGather.spec.ts

```diff
@@ -57,6 +57,7 @@ Parameters:
   )
   .params(u =>
     u
+      .beginSubcases()
```

Comment:
note: you can alternatively use `.paramsSubcasesOnly(u =>` instead of `.beginSubcases()` at the start, but it's equivalent so either is fine.

Reply:
done